sklearn/mixture/gmm.py

```diff
@@ -329,23 +326,42 @@ def score_samples(self, X):
         responsibilities = np.exp(lpr - logprob[:, np.newaxis])
         return logprob, responsibilities
 
-    def score(self, X, y=None):
-        """Compute the log probability under the model.
+    def score_samples(self, X):
+        """Return the per-sample log probability of the data under the model.
 
         Parameters
         ----------
         X : array_like, shape (n_samples, n_features)
-            List of n_features-dimensional data points.  Each row
+            List of n_features-dimensional data points. Each row
             corresponds to a single data point.
 
         Returns
         -------
         logprob : array_like, shape (n_samples,)
-            Log probabilities of each data point in X
+            Log probabilities of each data point in X.
         """
-        logprob, _ = self.score_samples(X)
+        logprob, _ = self._lp_resp_samples(X)
         return logprob
 
+    def responsibility_samples(self, X):
```

Comment:
what is this method for? Why not just use ``predict_proba``? We would rather not add to many functions to the API.